src/views/pages/GenerateKeypair/index.js

```diff
@@ -27,9 +35,23 @@ function GenerateKeypair() {
         <IdentitySubNav />
       </div>
       <div className="basis-3/4 px-8">
-        <PageTitle>Generate Key Pair</PageTitle>
-        <Text>{publicKey}</Text>
-        <Button onClick={generateKeypair}>Generate</Button>
+        <PageTitle>Generate Keypair</PageTitle>
+        <Text>{state.publicKey}</Text>
```

Comment:
I think this is where it's currently printing "asdf" for me - is that a placeholder or the response from the service?